<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>group_chat</class>
 <widget class="QWidget" name="group_chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">max-width: 450px;
max-height: 450px;
background-color: rgb(201, 228, 214)</string>
  </property>
  <widget class="QLabel" name="groupName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Group name</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chatWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 	rgb(211,211,211)</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 	rgb(211,211,211)</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupNameLine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLine">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>91</width>
     <height>231</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>229</height>
     </rect>
    </property>
    <widget class="QListWidget" name="userList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="privateChatButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 	rgb(211,211,211)</string>
   </property>
   <property name="text">
    <string>Private Chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="groupLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Imprint MT Shadow</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 132, 137)</string>
   </property>
   <property name="text">
    <string>GROUP CHAT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
